gpii/node_modules/contextManager/src/ContextManagerUtilities.js

```diff
@@ -0,0 +1,105 @@
+/*!
+GPII Context Manager Utilities
+
+Copyright 2014 Raising the floor - international
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+/* global require */
+"use strict";
+
+var fluid = fluid || require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    $ = fluid.registerNamespace("jQuery");
+
+fluid.registerNamespace("gpii.contextManager.utils");
+fluid.registerNamespace("gpii.contextManager.transforms");
+
+gpii.contextManager.utils.transformTypeLookup = {
+    "http://registry.gpii.net/conditions/inRange": "gpii.contextManager.transforms.inRange",
+    "http://registry.gpii.net/conditions/timeInRange": "gpii.contextManager.transforms.timeInRange"
+
+};
+
+// Transforms required for condition evaluation:
+fluid.defaults("gpii.contextManager.transforms.inRange", {
+    gradeNames: "fluid.standardTransformFunction"
+});
+
+gpii.contextManager.transforms.inRange = function (value, transformSpec) {
+    return (transformSpec.min && transformSpec.min > value) ||
+        (transformSpec.max && transformSpec.max < value) ? false : true;
+};
+
+// Transforms required for condition evaluation:
+fluid.defaults("gpii.contextManager.transforms.timeInRange", {
+    gradeNames: "fluid.standardTransformFunction"
+});
+
+
+gpii.contextManager.transforms.timeInRange = function (value, transformSpec) {
+    var isEarlier = gpii.contextManager.transforms.timeInRange.isEarlier;
+    var currentTime = {
+        hours: (new Date(value)).getHours(),
```

Comment:
Dangerous - this will automatically apply the server's timezone - should at least be flagged with a TODO. In general the standard libraries are inadequate for this and we will need to find a suitable utilities package